Libs/MRML/Logic/vtkMRMLColorLogic.h

```diff
@@ -39,6 +39,57 @@ class vtkMRMLColorTableNode;
 class VTK_MRML_LOGIC_EXPORT vtkMRMLColorLogic : public vtkMRMLAbstractLogic
 {
 public:
+  typedef struct
+    {
+      std::string CodeValue;
+      std::string CodingSchemeDesignator;
+      std::string CodeMeaning;
+
+      void StandardTerm() {
```

Comment:
Simply declare the struct contructor and rely on `std::string` being initialized to an empty string.
```
struct StandardTerm
{
  StandardTerm(){}

  StandardTerm(const std::string& value, const std::string& designator, const std::string& meaning)
    : CodeValue(value), 
      CodingSchemeDesignator(designator),
      CodeMeaning(meaning)
  {
  }
  [...]
};
```

No need for `typedef` and `void`
See http://www.cplusplus.com/doc/tutorial/structures/
See